superset/db_engine_specs/presto.py

```diff
@@ -287,7 +287,7 @@ def convert_dttm(
         if isinstance(sqla_type, types.Date):
             return f"DATE '{dttm.date().isoformat()}'"
         if isinstance(sqla_type, types.TIMESTAMP):
-            return f"""TIMESTAMP '{dttm.isoformat(timespec="microseconds", sep=" ")}'"""
+            return f"""TIMESTAMP '{dttm.isoformat(timespec="milliseconds", sep=" ")}'"""
```

Comment:
@villebro do you think this change should be in the `PrestoBaseEngineSpec` class (as it is currently proposed) or the `PrestoEngineSpec` class?

Reply:
@villebro @john-bodley checking in to see if this decision is blocking, or if we should just merge and revisit (strike while the CI is green)

Reply:
@rusackas I think it's likely blocking. @zhaorui2022 would you mind making the change in the `PrestoEngineSpec` and invoking the base class—via `super()`—where appropriate?

Reply:
@zhaorui2022 any interest in picking this up?